arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -5306,6 +5479,89 @@
         "modelAsString": false
       },
       "description": "Supported permission types."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Provisioning",
+        "Succeeded",
+        "Expiring",
+        "Deleting",
+        "HumanIntervention",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningStates",
+        "modelAsString": true
+      },
+      "description": "Provisioning state."
+    },
+    "ConnectorType": {
+      "type": "string",
+      "enum": [
+        "None",
+        "CRM",
+        "AzureBlob",
+        "Salesforce",
+        "ExchangeOnline",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "ConnectorTypes",
+        "modelAsString": true
+      },
+      "description": "Type of connector."
+    },
+    "SuggestRelationshipLinksResponse": {
+      "required": [
```

Comment:
if this is a response, wouldn't it make sense to rather mark stuff as `readOnly`?